src/NetEscapades.AspNetCore.SecurityHeaders/HeaderPolicyCollectionExtensions.cs

```diff
@@ -51,6 +51,9 @@ public static HeaderPolicyCollection AddDefaultSecurityHeaders(this HeaderPolicy
             builder.AddFrameAncestors().None();
         });
         policies.AddCrossOriginOpenerPolicy(x => x.SameOrigin());
+        policies.AddCrossOriginEmbedderPolicy(builder => builder.RequireCorp());
+        policies.AddCrossOriginResourcePolicy(builder => builder.SameOrigin());
```

Comment:
I think this one is probably ok for the document headers, because you're less likely to be calling it from _other_ sites (unless it's acting as a CDN or something)